docker-app/qfieldcloud/core/urls.py

```diff
@@ -105,4 +105,24 @@
         deltas_views.ListDeltasByDeltafileView.as_view(),
     ),
     path("deltas/apply/<uuid:projectid>/", deltas_views.ApplyView.as_view()),
+    path(
+        "organizations/<str:organization_name>/teams/",
+        users_views.TeamListCreateView.as_view(),
+        name="team_list_create",
+    ),
+    path(
+        "organizations/<str:organization_name>/teams/<str:team_name>/members/",
+        users_views.ListCreateTeamMembersView.as_view(),
+        name="team-member-list-create",
+    ),
+    path(
+        "organizations/<str:organization_name>/teams/<str:team_name>/members/<str:member_username>/",
+        users_views.GetUpdateDestroyTeamMemberView.as_view(),
+        name="team-member-retrieve-update-destroy",
+    ),
+    path(
+        "organizations/<str:organization_name>/teams/<str:team_name>/",
+        users_views.TeamDetailView.as_view(),
+        name="team_detail",
+    ),
```

Comment:
```suggestion
    path(
        "organizations/<str:organization_name>/teams/",
        team_views.ListCreateTeamView.as_view(),
        name="team_list_create",
    ),
    path(
        "organizations/<str:organization_name>/teams/<str:team_name>/",
        team_views.GetUpdateDestroyTeamView.as_view(),
        name="team_detail",
    ),
    path(
        "organizations/<str:organization_name>/teams/<str:team_name>/members/",
        team_views.ListCreateTeamMembersView.as_view(),
        name="team_member_list_create",
    ),
    path(
        "organizations/<str:organization_name>/teams/<str:team_name>/members/<str:member_username>/",
        team_views.GetUpdateDestroyTeamMemberView.as_view(),
        name="team_member_retrieve_update_destroy",
    ),
```

A bit of renaming here :)

Reply:
Done!